src/components/Table.vue

```diff
@@ -8,7 +8,7 @@
     />
 
     <table>
-      <caption>{{ tableTitle }}</caption>
+      <caption v-show="tableCaption">{{ tableCaption }}</caption>
```

Comment:
```suggestion
      <caption v-if="tableCaption">{{ tableCaption }}</caption>
```

Considering that the `tableCaption` can be null, it shouldn't even be rendered if there's no value.